packages/manager/modules/network-common/src/vrack/api/put/updateVrackServices.ts

```diff
@@ -1,5 +1,5 @@
 import { apiClient } from '@ovh-ux/manager-core-api';
-import { UpdateVrackServicesParams, VrackServices } from '@/types';
+import { UpdateVrackServicesParams, VrackServices } from '../../types';
 
 export const updateVrackServicesQueryKey = (vrackServicesId: string) => [
```

Comment:
it should be in a vrack-services folder, not in the vrack folder IMO since it's a different endpoint/resource for the API